boost-maven-plugin/pom.xml

```diff
@@ -1,166 +1,202 @@
+			<artifactId>plugin-support</artifactId>
+			<version>1.0-alpha-1</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.maven</groupId>
+					<artifactId>maven-project</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.maven</groupId>
+					<artifactId>maven-artifact</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.maven</groupId>
+					<artifactId>maven-plugin-api</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+	</dependencies>
 
-    <profiles>
-        <profile>
-            <id>it</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-invoker-plugin</artifactId>
-                        <version>3.0.1</version>
-                        <configuration>
-                            <debug>false</debug>
-                            <goals>
-                                <goal>install</goal>
-                            </goals>
-                            <projectsDirectory>src/it</projectsDirectory>
-                            <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
-                            <streamLogs>true</streamLogs>
-                        </configuration>
-                        <executions>
-                            <execution>
-                                <id>integration-test</id>
-                                <goals>
-                                    <goal>install</goal>
-                                    <goal>run</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
+	<profiles>
+		<profile>
+			<id>it</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-invoker-plugin</artifactId>
+						<version>3.0.1</version>
+						<configuration>
+							<debug>false</debug>
+							<goals>
+								<goal>install</goal>
+							</goals>
+							<projectsDirectory>src/it</projectsDirectory>
+							<cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
+							<streamLogs>true</streamLogs>
+							<properties>
+                                <runtimeGroupId>${runtimeGroupId}</runtimeGroupId>
+                                <runtimeArtifactId>${runtimeArtifactId}</runtimeArtifactId>
+                                <runtimeVersion>${runtimeVersion}</runtimeVersion>
+                            </properties>
+						</configuration>
+						<executions>
+							<execution>
+								<id>integration-test</id>
+								<goals>
+									<goal>install</goal>
+									<goal>run</goal>
+								</goals>
+							</execution>
+						</executions>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
 
```

Comment:
Would be real nice if we could default from the cmd line so we didn't have to specify ` -DruntimeVersion=18.0.0.3 -Druntime=ol`.    I realize it's more work to add the logic, so could be a follow-up issue.

Reply:
18.0.0.3 is less compatible with Java 9 than 18.0.0.3, due to preparations to support Java 11 in 18.0.0.4. As a result, we don't test 18.0.0.3 on Java 9 in LMP. I think we should do the same thing here.

Reply:
This is added in 2a75aaa8b400ceda0e1aec9babffc86acbebf066.